### PR TITLE
Wendy G Bite HW 1.3

### PR DESCRIPTION
**Pull request by AT89**
Comments: I couldn't get the fonts to fit correctly, before i resized the picture, they were somewhat aligned right. I noticed that the picture is off compared to the example. Also need to look into how the borders between bottom links were done (mine are too big). will look at solutions later tonight.

4.9/5 completeness
3/5 comfort level

### CODE REVIEW

File: resume.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+</header>
+
+
+<h2>Resume</h2>
+
+<p class=resume>I recently graduated from a Web Development course at General Assembly where I increased my typing speed from 10 words per minute on my type writer to 100 words per minute on my new MacBook Pro. They also taught me how to be an <strong>awesome</strong>Web Developer. During my spare time, I enjoy senior water aerobics, pickling, and spending time with my great grand children.
+</p>
+<h3>Relevant Experience</h3>
+
+<li><titlee>Web Developer</titlee>GitHub 2013 - present
```
Comment (RobertAKARobin) [inline resume.html]: What is `<titlee>`? Don't just invent your own HTML tags. Also, `<title>` only goes inside your `<head>`. There will only ever be one `<title>` on a page.

File: resume.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+<li>
+<titlee>Senior Vice</titlee> President The Golden Girls Appreciation Society 1985 - 1992
+Managed a team of three and over saw day to day operations at The Golden Girls Retirement Facility in Miami.
+</li>
+
+<li>
+Certified Mint Candy and Freshly Baked Cookies Distributor. 1960S
+Won multiple awards for my secret chocolate chip recipies. My grand children enjoy them too :)
+</li>
```
Comment (RobertAKARobin) [inline resume.html]: `<li>` must go inside a `<ol>` or `<ul>`.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  font-weight: bold;
+}
+.active2{
+  color: #FFFFFF;
+  text-decoration: none;
+  font-weight: bold;
+}
+nav a.active2:before{
+  color: #777777;
+  content: " | "
```
Comment (RobertAKARobin) [inline style.css]: Noice!

**Comment by RobertAKARobin** [comment]
You're doing great with CSS! You're a bit rusty on HTML, though. Next time you do *anything* in HTML, run it through the HTML validator:

https://validator.w3.org/

It will show you any errors you've made.

✅

**Comment by AT89** [comment]
Hey Robin,

hope you had a great weekend. Thanks for the review. I put the titlee thing
in to test out 'custom tags' but I guess I forgot to re-edit it (shouldve
branched) after the time mark.

I was trying to have the text different than the other lines on the
"titlee" but I should have subbed. Actually Jesse showed me that custom
tags could still work. I need to look back into it.

Is it bad practice to use custom tags? I figured it would solve the task
similar to using sub tags.

On Fri, May 27, 2016 at 5:54 PM, Robert Thomas <notifications@github.com>
wrote:

> You're doing great with CSS! You're a bit rusty on HTML, though. Next time
> you do *anything* in HTML, run it through the HTML validator:
>
> https://validator.w3.org/
>
> It will show you any errors you've made.
>
> ✅
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ga-wdi-exercises/wendy_bite/pull/66#issuecomment-222261665>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASTlsi71PVbCltwhncwxZ7exDx5_XbIxks5qF2f9gaJpZM4InFaz>
> .
>



-- 
Anh "Ian" T. Truong
Brother, Alpha Kappa Psi Fraternity, Inc.
Alumni, Christopher Newport University
anh.truong.07@cnu.edu

**Comment by RobertAKARobin** [comment]
Yes, it's bad practice. Other browsers may not be able to interpret them properly. It's just like inventing a word in a spoken language. `<sub>` isn't a tag I invented -- it's a standard HTML tag. You can see the list here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element

**Comment by RobertAKARobin** [comment]
I'm going to close this just to remind myself that I've already checked this homework. I should have closed it when I actually did check it, but forgot. You're welcome to keep replying!

**Pull request closed without merging**